src/main.rs

```diff
@@ -26,7 +30,17 @@ async fn main() -> anyhow::Result<()> {
     let args = Args::parse();
 
     let client = Client::try_default().await?;
-    let pod_list = args.pods;
+
+    let mut pod_list = args.pods;
+
+    if args.deployments.len() > 0 {
```

Comment:
🚫 **[clippy]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> 
<pre><code>error: length comparison to zero
  --> src/main.rs:36:8
   |
36 |     if args.deployments.len() > 0 {
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^ help: using `!is_empty` is clearer and more explicit: `!args.deployments.is_empty()`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_zero
   = note: `-D clippy::len-zero` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::len_zero)]`
</code></pre>
<!-- __reviewdog__:ChBhZWU2NmI0MmE5MmM0ZWFjEgZjbGlwcHk= -->